### DISCUSSION TITLE
Is there a method to prefix page numbers with another number?

### DISCUSSION THREAD

**Discussion by entelecheia1963** [quarto-dev/quarto-cli#7712] [Q&A] [answered]
### Description

I am developing a set of student workbooks using Quarto to render from Python / Jupyter into PDF documents. I have many of these workbooks. 

I would like to be able to pre-pend page numbers with my own defined prefix. Specifically .. If I am publishing workbook '4', then I would like pagenumbers to show as:

4-1, 4-2, 4-3 .... etc.

Another way of saying this is that, I am now defining the numbering for my whole workbook using:

````
include-in-header:
  - text: |
      \setcounter{section}{3}
````

It would be ideal to pre-pend that same section number to each page number.

**Comment by mcanouil** [comment]
This is a LaTeX question that you could find easily the answer online.

**Comment by entelecheia1963** [comment]
Thank you for your help as always Mcanouil. I don't know anything about Latex. I don't now how knowing a latex solutions would map onto knowing how to do it using Quarto. I.e. if I had a latex solution - what would that mean for me working in Quarto? 

I think your assumption may be that I know all about the tool chain here and about each of its elements. I don't. I am just using Quarto to render PDFs - and I want to find out how to change the page numbers. 

If there is an answer that is easy to find .. I could really do with some help in finding it please.

**Reply by mcanouil** [reply, marked as answer]
You already are using LaTeX commands that you insert ion Quarto, so not an assumption.
Now, if you are using commands you don't understand, the issue is elsewhere.

```yaml
include-in-header:
  - text: |
      \setcounter{section}{3}
```

If you want to do advanced PDF/LaTeX I am afraid, you'll have to learn at least a bit of LaTeX.

See <https://duckduckgo.com/?q=change+page+number+latex> which will give you LaTeX commands that you can insert (see <https://quarto.org/docs/reference/formats/pdf.html#includes>).

Note that we cannot teach users LaTeX, Open XML, HTML, CSS, JavaScript, Typst, etc.
If Quarto (Pandoc) default template does not fit your needs, I am afraid you're gonna have to learn more than markdown.

```latex
\renewcommand{\thepage}{3-\arabic{page}}
```

**Reply by entelecheia1963** [reply]
Ah yes! That appears to work. Thank you
